### DISCUSSION TITLE
Buy [REDACTED] domain

### DISCUSSION THREAD

**Discussion by Myzel394** [schlagmichdoch/PairDrop#14] [Ideas]
Hey I just found out there's a domain available that would perfectly fit to this project. It's not expensive (about 20€ per year) and it has something to do with PairDrop / AirDrop. I don't want to write the domain here publicly because there will probably be bots or people around here waiting to claim such names. I don't think I can write direct messages here on GitHub, can't I? You could write me an email (my email is in my bio) or type your email here. Or you can reach out to me via Mastodon or Twitter, or of course any other communication channel you prefer :D

**Comment by schlagmichdoch** [comment]
Just send me an email: sonnig-02.hieven@icloud.com

**Comment by Myzel394** [comment]
@schlagmichdoch I sent you a mail
